iron-doc-behavior.html

```diff
@@ -150,11 +171,16 @@
         return element.classname || element.tagname;
       },
 
-      _compareElements: function(a, b) {
-        if (a.tagname === b.tagname) {
-          return a.classname == null ? -1 : a.classname.localeCompare(b.classname);
-        }
-        return a.tagname == null ? -1 : b.tagname == null ? 1 : a.tagname.localeCompare(b.tagname);
+      /**
+       * Compare two analysis descriptors (elements, functions, etc.) by
+       * display name.
+       */
+      _compareDescriptors: function(a, b) {
+        // Elements display as "<tagname> (classname)" or "classname", while
+        // everything else displays as "name". Sort according to that format.
+        var aName = (a.name || '') + (a.tagname || '') + (a.classname || '');
+        var bName = (b.name || '') + (b.tagname || '') + (b.classname || '');
```

Comment:
Latest commit has a version that I think is closer? Elements with no tagname will sort earlier, which in at least one one example worked out better and is easier to write (Polymer.Element came first, and that seems important).